Examples/HackerNewsClient/HackerNewsClient/UI/Browser.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Browser.swift
+//  HackerNewsClient
+//
+//  Copyright © 2016 Bending Spoons.
+//  Distributed under the MIT License.
+//  See the LICENSE file for more information.
+
+import Foundation
+import Katana
+import KatanaElements
+
+struct Browser: NodeDescription, PlasticNodeDescription {
```

Comment:
I was thinking about the first approach and have some doubts on how to proceed. My first guess would be to implement a `LoadURLAction` (an async action with side effect) in which I would make the web view start loading and keep track of the progress until the end.

My doubts are:
1. This would require adding a `webViewLoading: Bool` field in the app state `HackerNewsState`, right? Isn't there a better alternative to keep this internal to the `WebView` component?
2. Where should I dispatch such an action and with which payload (I would need the native `WKWebView` and this doesn't feel clean enough).